lib/isDataUri.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = isDataUri;
+
+/**
+ * @private
+ */
+ function isDataUri(uri) {

```

Comment:
What happens if `uri` is undefined?  We could throw an exception, but I think it would also be OK to just make sure this returns `false`.